samples/.drone.yml

```diff
@@ -48,15 +49,17 @@ steps:
     # "Executing Snakemake workflow."
     - snakemake
 
-    - mkdir "$DRONE_BUILD_NUMBER"
+    #- mkdir "$DRONE_BUILD_NUMBER"
     #- git status | grep modified | awk '{print $2}'
     #- git diff-tree -r --no-commit-id --name-only origin/master . | xargs -I '{}' cp '{}' "$DRONE_BUILD_NUMBER"/
 
     # "[+] Back-Pushing output."
+    - git stash
```

Comment:
Why stash?  If the pipeline edits any tracked files, this will discard the changes, which is the output we want.